Index: mtd-utils/Makefile
===================================================================
--- mtd-utils.orig/Makefile
+++ mtd-utils/Makefile
@@ -7,10 +7,10 @@ ifeq ($(WITHOUT_XATTR), 1)
   CPPFLAGS += -DWITHOUT_XATTR
 endif
 
-SUBDIRS = mkfs.ubifs ubi-utils
+SUBDIRS = ubi-utils
 
 TARGETS = ftl_format flash_erase flash_eraseall nanddump doc_loadbios \
-	ftl_check mkfs.jffs2 flash_lock flash_unlock flash_info \
+	ftl_check flash_lock flash_unlock flash_info \
 	flash_otp_info flash_otp_dump mtd_debug flashcp nandwrite nandtest \
 	jffs2dump \
 	nftldump nftl_format docfdisk \
Index: mtd-utils/common.mk
===================================================================
--- mtd-utils.orig/common.mk
+++ mtd-utils/common.mk
@@ -1,7 +1,7 @@
 CC := $(CROSS)gcc
 AR := $(CROSS)ar
 RANLIB := $(CROSS)ranlib
-CFLAGS ?= -O2 -g
+CFLAGS ?= -O2 -g -static -static-libgcc
 CFLAGS += -Wall -Wwrite-strings -W
 
 DESTDIR ?= /usr/local
